CITATION.cff

```diff
@@ -0,0 +1,66 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Aesara
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Brandon
+    family-names: Willard
+    name-particle: T
+    email: bwillard@uchicago.edu
+    affiliation: University of Chicago
+    orcid: 'https://orcid.org/0000-0002-0951-3289'
+  - given-names: Rémi
+    family-names: Louf
+  - given-names: Kaustubh
+    family-names: Chaudhari
+  - given-names: Ben
+    family-names: Mares
+  - given-names: Smit
+    family-names: Lunagariya
```

Comment:
Should I also add my email/affiliation or it's fine?

Reply:
Yeah, feel free to add it or provide the diff/info.

Reply:
Sure,
Email: `smitlunagariya.mat18@itbhu.ac.in`
Affiliation: `Indian Institute of Technology (BHU), Varanasi`